views/check-your-answers.html

```diff
@@ -9,606 +9,202 @@
+              text: "Principal or registered office address"
+            },
+            value: {
+              html: formattedPrincipalAddressHtml
+            }
+          },
+          {
+            key: {
+              text: "Correspondence address"
+            },
+            value: {
+              html: formattedServiceAddressHtml
+            }
+          },
+          {
+            key: {
+              text: "Email"
+            },
+            value: {
+              text: appData.entity.email
+            }
+          },
+          {
+            key: {
+              text: "Legal Form"
+            },
+            value: {
+              text: appData.entity.legalForm
+            }
+          },
+          {
+            key: {
+              text: "Governing law"
+            },
+            value: {
+              text: appData.entity.governedLaw
+            }
+          },
+          {
+            key: {
+              text: "Overseas public register"
+            },
+            value: {
+              text: appData.entity.publicRegister
+            }
+          },
+          {
+            key: {
+              text: "Registration Number"
+            },
+            value: {
+              text: appData.entity.registrationNumber
+            }
+          }
+        ]
+      }) }}
 
-        {{ govukCheckboxes({
+      {{ govukCheckboxes({
         idPrefix: "due-diligence",
         name: "due-diligence",
         fieldset: {
-        legend: {
-        text: "Due diligence",
-        isPageHeading: false,
-        classes: "govuk-fieldset__legend--m"
-        }
+          legend: {
+            text: "Due diligence",
+            isPageHeading: false,
+            classes: "govuk-fieldset__legend--m"
+          }
         },
         hint: {
-        text: "Companies House has processes in place to check the accuracy of the information you provide in this service."
+          text: "Companies House has processes in place to check the accuracy of the information you provide in this service."
         },
         items: [
-        {
-        value: "due-diligence-1",
-        text: "I confirm that due diligence has been done to confirm the accuracy of all the information provided."
-        },
-        {
-        value: "due-diligence-2",
-        text: "I confirm that all beneficial owners and managing officers of the overseas entity will be informed of their legal responsibility to keep their information up to date."
-        }
+          {
+            value: "due-diligence-1",
+            text: "I confirm that due diligence has been done to confirm the accuracy of all the information provided."
+          },
+          {
+            value: "due-diligence-2",
+            text: "I confirm that all beneficial owners and managing officers of the overseas entity will be informed of their legal responsibility to keep their information up to date."
+          }
         ]
+      }) }}
+
+      <form method="post">
```

Comment:
I can not see the POST on check your answers controller. We need to include it as well

Reply:
I was going to say this wasn't part of the story but it looks like it's changed since I've started working on it?

Reply:
It is fine I will add it in a separate PR with the transaction calls

Reply:
Dan has confirmed he doesn't the button to do anything yet for this story so no need for a POST method just yet